default_mulesoft_traceability_agent.yml

```diff
@@ -17,8 +17,8 @@ apic_traceability_agent:
     auth:
       clientId: "${CENTRAL_AUTH_CLIENTID}"
       keyPassword: ${CENTRAL_AUTH_KEYPASSWORD:""}
-      privateKey: "${CENTRAL_AUTH_PRIVATEKEY:/keys/private_key.pem}"
-      publicKey: "${CENTRAL_AUTH_PUBLICKEY:/keys/public_key.pem}"
+      privateKey: "${CENTRAL_AUTH_PRIVATEKEY:}"
+      publicKey: "${CENTRAL_AUTH_PUBLICKEY:}"
```

Comment:
The : can be dropped since you are not setting a default value

Reply:
I wanted to keep the default config files as it is if a developer was to test it by building a binary, they can refer to this.